### ISSUE TITLE
it = torch.exp(torch.matmul(self.wi, x) + self.bi)

### ISSUE THREAD

**Issue by Tomcat099** [andrewgcodes/xlstm#1]
it = torch.exp(torch.matmul(self.wi, x) + self.bi)
        ft = torch.sigmoid(torch.matmul(self.wf, x) + self.bf)
        
        在it和fi的计算中，wi*x的过程中不应该是wi的T(即转置)和x相乘嘛？wf同理，在代码实现中好像没有体现出来，可以解答下嘛？
